### PR TITLE
Blueprints

### PR DESCRIPTION
**Pull request by kovshenin**
Blueprints are coming soon to a sailboat near you.

Blueprints will allow Sail users to specify a manifest YAML file, which declares which plugins and themes should be installed and activated, additional server software setup and configuration and more. Work in progress will be committed and tested on this branch, API changes will be deployed to the production API service.

### CODE REVIEW

**Comment by soulseekah** [comment]
Beautiful.

File: sail/blueprints/sample.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+version: 1
+
+# Install plugins from the WordPress.org repo or custom sources.
+plugins:
+  woocommerce: latest
```
Comment (DimaMinka) [inline sail/blueprints/sample.yaml:6]: - [feat] plugin auto activation option

File: sail/blueprints/sample.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  wordpress-seo: latest
+  hello-dolly:
+    url: https://downloads.wordpress.org/plugin/hello-dolly.1.7.2.zip
+
+# Install themes from the WordPress.org directory or custom sources.
+themes:
```
Comment (DimaMinka) [inline sail/blueprints/sample.yaml:12]: [feat] theme activation option on deploy

File: sail/blueprints/sample.yaml (hunk 1 above)
Comment (kovshenin) [reply sail/blueprints/sample.yaml:6]: @DimaMinka by default all plugins specified in a blueprint will be activated after install. Is there a good use case where that would not be ideal?

File: sail/blueprints/sample.yaml (hunk 2 above)
Comment (kovshenin) [reply sail/blueprints/sample.yaml:12]: Last theme in the list is activated by default :)

**Comment by DimaMinka** [comment]
interesting... wait to test...

**Pull request merged**